api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -57,15 +58,17 @@ public class InstitutionServiceTest {
       new Institution()
           .shortName("test")
           .displayName("this is a test")
-          .organizationTypeEnum(OrganizationType.INDUSTRY);
+          .organizationTypeEnum(OrganizationType.INDUSTRY)
+          .bypassInitialCreditsExpiration(false);
```

Comment:
arbitrary - it just has to be set to something non-null to pass round trip checks.